README.md

```diff
@@ -58,8 +58,11 @@ In order to run the tests, you'll need to install the dependencies first. The ch
 # Uncomment if you don't have poetry installed
 # pip install poetry
 
+# Activate the environment
+poetry shell
+
 # Install the dependencies
-poetry install --with dev
+poetry install
```

Comment:
I think there was a specific reason why we added `--with-dev` here. If the `poetry shell` fixes those issues this is fine. But I remember both @TimoGlastra and me running into issues with `poetry install` instead of `poetry install --with-dev`.

Reply:
--with dev is not an allowed option in the poetry version i have 😅

Reply:
```

  NoSuchOptionException

  The "--with" option does not exist.
```

Reply:
ah I had poetry <1.2.0 

upgrade to > 1.2.0 - now that's an option